src/routing/comm.rs

```diff
@@ -9,80 +9,76 @@
 use crate::error::{Error, Result};
 use bytes::Bytes;
 use err_derive::Error;
-use futures::{
-    lock::Mutex,
-    stream::{FuturesUnordered, StreamExt},
-};
-use lru_time_cache::LruCache;
+use futures::stream::{FuturesUnordered, StreamExt};
 use qp2p::{Connection, Endpoint, QuicP2p};
-use std::{future::Future, net::SocketAddr, sync::Arc};
-use tokio::{
-    sync::{mpsc, watch},
-    task,
+use std::{
+    fmt::{self, Debug, Formatter},
+    net::SocketAddr,
 };
-
-// Number of Connections to maintain in the cache
-const CONNECTIONS_CACHE_SIZE: usize = 1024;
-
-/// Maximal number of resend attempts to the same target.
-pub const RESEND_MAX_ATTEMPTS: u8 = 3;
```

Comment:
just to confirm, the all these (cache & resend) are moved into QuicP2p ?

Reply:
The cache yes. The resend isn't explicitly implemented in qp2p, but according to @dirvine , quinn/quic should do something like that by itself.

Reply:
ok then.